app/controllers/v1/ControllerUtils.scala

```diff
@@ -43,14 +43,16 @@ trait ControllerUtils extends Controller with StrictLogging with Properties with
   protected type UnitLinksListType = Seq[JsValue]
   protected type StatisticalUnitLinkType = JsValue
 
-  private def toJson(record: (JsValue, JsValue), `type`: String): JsValue = {
+  private def toJson(record: (JsValue, JsValue), `type`: String, periodFromRowKey: String): JsValue = {
     Json.toJson(JsObject(record match {
       case (link, unit) =>
 
         // @ TODO PATCH - fix and remove patch
         // BI does not have period, so use an empty string
         val period = if (`type` == ENT.toString) {
           (unit \ "period").getOrNull
+        } else if (`type` == LOU.toString) {
```

Comment:
I will do that in future, there was already an if statement for handling different unit types so since a future refactor is not far off I thought I'd follow the same format.